src/support/bits.cpp

```diff
@@ -165,6 +163,10 @@ uint32_t Log2(uint32_t v) {
       return 4;
     case 32:
       return 5;
+    default: {
+      assert(IsPowerOf2(v));
+      return 31 - CountLeadingZeroes(v);
```

Comment:
this changes the behavior. previously an invalid value would abort (due to the `WASM_UNREACHABLE`). With this change, it just asserts in builds with assertions enabled, so a release build will ignore the issue.

I think we can remove the switch entirely. Then check if it's a power of two and abort if not. Then return the `31 - ` computation.

Reply:
done!